frontstage/views/surveys.py

```diff
@@ -241,6 +241,9 @@ def upload_survey(session):
         return render_template('surveys/surveys-upload-success.html', _theme='default', upload_filename=upload_filename)
     else:
         logger.debug('Upload failed')
-        error_info = json.loads(result.text)
+        if result.text is None:
+            logger.error('Upload failed due to error')
+        else:
+            error_info = json.loads(result.text)
         return render_template('surveys/surveys-upload-failure.html',  _theme='default', error_info=error_info,
```

Comment:
will this render_template work if no error_info has been defined?, i.e. if result.text=None